app/views/cancelation_mailer/cancelation_confirmation.text.erb

```diff
@@ -4,3 +4,5 @@
 
 <%= t("mailer.cancelation_mailer.cta") %>
 <%= proposal_url(@proposal) %>
+
+<%= t("mailer.footer", feedback_url: feedback_url) %>
```

Comment:
this is the text-only email so it doesn't use the HTML template layout -- is there a separate layout for text emails?